tests/attested_tls/common/gencreds.cpp

```diff
@@ -0,0 +1,272 @@
+// Copyright Microsoft. 
```

Comment:
I did not find reference to the function generate_attested_credentials. Do you need this file?

Reply:
This func is been used in "_configure_srv" in oe_enclave tls server. Do you mean, I should use the certificate which mbedtls provided?